camera-core/src/main/java/com/stripe/android/camera/scanui/SimpleScanStateful.kt

```diff
@@ -31,12 +31,16 @@ internal interface SimpleScanStateful<State : ScanState> {
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class ScanErrorListener : AnalyzerLoopErrorListener {
     override fun onAnalyzerFailure(t: Throwable): Boolean {
-        Log.e(Config.logTag, "Error executing analyzer", t)
+        Log.e(TAG, "Error executing analyzer", t)
         return false
     }
 
     override fun onResultFailure(t: Throwable): Boolean {
-        Log.e(Config.logTag, "Error executing result", t)
+        Log.e(TAG, "Error executing result", t)
         return true
     }
+
+    private companion object {
+        val TAG: String = ScanErrorListener::class.java.simpleName
```

Comment:
this should probable be `SimpleScanStateful`

Reply:
done